web/styles/app_variables.css

```diff
@@ -98,6 +98,7 @@
     --navbar-popover-menu-min-width: 230px;
     --message-actions-popover-min-width: 230px;
     --topic-actions-popover-min-width: 200px;
+    --user-card-popover-min-width: 200px;
```

Comment:
I have started out with a min-width to be `200px` so that the names don't
look too compressed, but we would likely need to discuss this in CZO to
come with a min-width that has a better reasoning behind it.